src/ScriptBuilderTask.Tests/InnerTaskTests.cs

```diff
@@ -12,11 +11,11 @@ public void IntegrationTest()
     {
         var testDirectory = TestContext.CurrentContext.TestDirectory;
         var temp = Path.Combine(testDirectory, "InnerTaskTemp");
-        if (!Directory.Exists(temp))
+        if (Directory.Exists(temp))
         {
-            return;
```

Comment:
I'm confused by this change. Was this broken before? It sems like it was bailing out of the test early if the temp folder was missing, is that right?

Reply:
yes, this was incorrect. On the first, unless you create that directory manually the test would not generate anything.